models/shifts/shift.py

```diff
@@ -0,0 +1,12 @@
+from .shifts_enum import ShiftsEnum
+
+
+class Shift:
+
+    def __init__(self, shift_type: ShiftsEnum):
+        self.type = shift_type
+        self.employee = None
```

Comment:
why arent shifts generated once its clear what they will contain? dont like classes that change too much after being created